src/controllers/cohort.js

```diff
@@ -51,3 +52,37 @@ export const getCohortById = async (req, res) => {
     return sendMessageResponse(res, 500, 'Unable to get cohort')
   }
 }
+
+export const getAllCohortExercises = async (req, res) => {
+  const cohortId = Number(req.params.id)
+  const whereData = {}
+  whereData.cohortId = cohortId
+
+  try {
+    const foundExercise = await CohortExercise.findAll({ whereData })
+
+    const formattedExercise = foundExercise.map((exercise) => {
+      return {
+        ...exercise.toJSON().exercise
+      }
+    })
+    return sendDataResponse(res, 200, { cohortExercises: formattedExercise })
+  } catch (e) {
+    return sendMessageResponse(res, 500, 'Unable to get exercise')
+  }
+}
+
+export const createCohortExercise = async (req, res) => {
+  const { exerciseId: exerciseID, cohortId: cohortID } = req.body
+
+  try {
+    const createdCohortExercise = await CohortExercise.createdCohortExercise(
```

Comment:
This looks like a typo - should it not be `createCohortExercise`, not `createdCohortExercise`?

Reply:
corrected